app/javascript/lib/scp-api.js

```diff
@@ -200,8 +200,11 @@ export function stringifyQuery(paramObj, sort) {
  *
  * @param {String} studyAccession Study accession
 */
-export async function fetchStudyFileInfo(studyAccession, mock=false) {
-  const apiUrl = `/studies/${studyAccession}/file_info`
+export async function fetchStudyFileInfo(studyAccession, includeOptions=true, mock=false) {
```

Comment:
Remind me -- does `file_info` leverage our Rails file cache?  

Maybe it's just because I'm on a development environment, but about 1/3 of my poll requests have > 5 s TTFB.   Even though it's used to assess current status of an occasionally-rapidly-changing object on the server, those objects will only be changing -- i.e. ingesting / parsing -- for a very brief phase of their life.  Cache updates would be rare, and not a notable performance burden in the parsing phase.

So caching seems like it'd be a big scalability improvement, if not done already.